firmware/Core/Inc/temperature_sensor/temperature_sensor.h

```diff
@@ -0,0 +1,18 @@
+#ifndef __INCLUDE_GUARD_TEMPERATURE_SENSOR_H__
+#define __INCLUDE_GUARD_TEMPERATURE_SENSOR_H__
+
+#include <stdint.h>
+
+typedef enum  
+{
+    ONE_BIT_PRECISION = 0,
```

Comment:
can you prepend all of these with TEMPERATURE_SENSOR_

Reply:
Fixed.